### ISSUE TITLE
[BUG] Microsoft Teams actions Information is missing in Actions

### ISSUE THREAD

**Issue by deepikakeshav-qasource** [elastic/security-docs#736]
**Description**: Microsoft Teams actions Information is missing in Actions

**URL link or topic name**: 
- https://www.elastic.co/guide/en/security/current/rules-ui-create.html

**Screenshots**:
![Actions_doc](https://user-images.githubusercontent.com/61860752/120311735-77f12300-c2f5-11eb-976d-775e19a44f46.jpg)

![actions_kibana](https://user-images.githubusercontent.com/61860752/120311736-77f12300-c2f5-11eb-96f7-925401c6cd1b.jpg)

**Comment by deepikakeshav-qasource** [comment]
@manishgupta-qasource Please Review!!

**Comment by manishgupta-qasource** [comment]
Reviewed & Assigned to @nastasha-solomon

**Comment by nastasha-solomon** [comment]
Thanks for filing this @deepikakeshav-qasource ! After some additional review, I think some additional changes also need to be made. Let me know what you think.

- Will need to fix the following in the [Set up alert notifications (optional)](https://www.elastic.co/guide/en/security/current/rules-ui-create.html#rule-notifications) section:
  - Change all instances of “action type” to “connector type” since "action type" is the term used in the Security app UI. 
  - Add to step 2 that users can also select to send notifications via MS Teams.  
- Will need to fix the following in the [Create Rule](https://www.elastic.co/guide/en/security/7.x/rules-api-create.html) topic:  
  - Change all instances of “action type” to “connector type”.
  - Add to the text beneach the code example that users can also select to send notifications via MS Teams.  

TIA for your help!

**Comment by deepikakeshav-qasource** [comment]
Hi @nastasha-solomon

> * Will need to fix the following in the [Set up alert notifications (optional)](https://www.elastic.co/guide/en/security/current/rules-ui-create.html#rule-notifications) section:
>   * Change all instances of “action type” to “connector type” since "action type" is the term used in the Security app UI.
>   * Add to step 2 that users can also select to send notifications via MS Teams.

> * Will need to fix the following in the [Create Rule](https://www.elastic.co/guide/en/security/7.x/rules-api-create.html) topic:
>   * Change all instances of “action type” to “connector type”.
>   * Add to the text beneath the code example that users can also select to send notifications via MS Teams.

Yes, It should be updated accordingly.

Please let us know when the changes are available. We will approve the PR accordingly.

Thanks!!

**Comment by nastasha-solomon** [comment]
deepikakeshav-qasource thanks for verifying! I've updated the image you flagged in your initial comment and included the additional changes I described. The only updates I _didn't_ make were to the note under step 2 [here](https://www.elastic.co/guide/en/security/current/rules-ui-create.html#rule-notifications) and the sentence "Each action type requires a connector" in the [Create Rule](https://www.elastic.co/guide/en/security/7.x/rules-api-create.html) topic. Before I finish my chagnes to these toipcs, I need to check with our UX and dev folks whether the correct term to use here is "action", "action type", or "connector type".
